app/src/utils/md.ts

```diff
@@ -1,15 +1,26 @@
 import { marked } from 'marked';
 import dompurify from 'dompurify';
 
+const renderer = new marked.Renderer();
+renderer.link = function (href, title, text) {
+	const link = marked.Renderer.prototype.link.apply(this, [href, title, text]);
+	return link.replace('<a', "<a target='_blank'");
```

Comment:
```suggestion
	return link.replace('<a', '<a target="_blank"');
```

tiny nitpick, but that's what i'm here for